sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -134,6 +134,17 @@ export interface EventHubConsumerOptions {
    * If no value is provided here, the retry options set when creating the `EventHubClient` is used.
    */
   retryOptions?: RetryOptions;
+  /**
+   * @property
+   * Indicates whether or not the consumer requests metrics for the state of its
+   * partition each time that events are received.
+
+   * When metrics are requested, each event received from the Event Hubs service will carry metadata
+   * about the state of a partition that it otherwise would not.  This results in a small amount of
+   * additional network bandwidth consumption that is generally a favorable trade-off when considered
+   * against periodically making requests for partition properties using the Event Hub client.
+   */
```

Comment:
For docs, please see the PR https://github.com/Azure/azure-sdk-for-net/pull/7505 where I have made fixes to the .Net code